bench/wtperf/runners/checkpoint_stress_schema_ops.wtperf

```diff
@@ -0,0 +1,19 @@
+# A stress configuration, to create long running checkpoints and see how
+# they interfere with schema level operations (table create, drop).
+# Setup a cache size large enough that checkpoints can take a long time.
+conn_config="cache_size=8GB,log=(enabled=false),checkpoint=(wait=30)"
+table_config="leaf_page_max=4k,internal_page_max=16k,type=file"
+icount=10000000
+table_count_idle=100
+# Turn on create/drop of idle tables, and error if a single operation takes
+# more than 5 seconds.
+idle_table_cycle=5
+populate_threads=5
+checkpoint_threads=0
```

Comment:
The wtperf checkpoint functionality pre-dates the WiredTiger library checkpoint thread. The only time the wtperf functionality is necessary is if you want multiple threads checkpointing, which is more interesting for stress testing than benchmarking. I'd be happy to remove the functionality from wtperf.